tests/expressions_case/tests.py

```diff
@@ -21,6 +22,7 @@
     Image = None
 
 
+@ignore_warnings(category=RemovedInDjango20Warning)
```

Comment:
Is this needed? `python -Wall tests/runtests.py expressions_case` doesn't give any problems without it? I didn't check the other ignores but maybe you can do that.